### DISCUSSION TITLE
How can I use function of Faker.js

### DISCUSSION THREAD

**Discussion by brkzkn** [mockapi-io/docs#10] [Q&A]
Hello, 

I didn't see any documentation about how can I use function of faker.js? For example, the response returns successfully when I use function name like `$random.alphaNumeric` but if I want to pass the parameter to that function, the response doesn't return. like `$random.alphaNumeric(10)`

Ref; https://fakerjs.dev/api/string.html#alphanumeric

![mock_io](https://github.com/mockapi-io/docs/assets/1784806/4d427455-cdaa-45ec-b6d5-0834c1d4ccf2)

**Comment by nsmirn0v** [comment]
Custom params are not yet supported

**Reply by sonal-farmart** [reply]
is it something in scope? i really want to use customisations
